wikicurses/main.py

```diff
@@ -35,6 +37,111 @@ def tabComplete(text, matches):
     return match
 
 
+def showImages():
+    """
+    Launch program to display the images for the current article.
+
+    The program is specified in the config, along with it's args.
+    Otherwise, the default is `feh`.
+    """
+    if 'wikipedia.org' not in wiki.siteurl:
+        ex.notify('Cancelled: Only works on Wikipedia')
+        return
```

Comment:
I think I tried it on vimwiki (it does have some images.) I can't recall the others. I've tested it on a few Wikia sites just now though and it seems I was too hasty in limiting it, as they are just fine.